configuration/configurator/ConfiguratorMain.cpp

```diff
@@ -70,12 +70,12 @@ void usage()
     char pBuildSetFile[BUFF_SIZE];
```

Comment:
Perhaps outside of the scope of this issue, but are these other char[BUFF_SIZE] candidates to be converted to StringBuffers as well?